packages/astro/src/core/errors/errors-data.ts

```diff
@@ -1074,8 +1064,8 @@ export const ConfigNotFound = {
 export const ConfigLegacyKey = {
 	name: 'ConfigLegacyKey',
 	title: 'Legacy configuration detected.',
-	message: (legacyConfigKey: string) => `Legacy configuration detected: \`${legacyConfigKey}\`.`,
-	hint: 'Please update your configuration to the new format.\nSee https://astro.build/config for more information.',
+	message: (legacyConfigKey: string) =>
+		`Legacy configuration detected: \`${legacyConfigKey}\`. update your configuration to the new format.`,
```

Comment:
```suggestion
		`Legacy configuration detected: \`${legacyConfigKey}\`. Update your configuration to the current supported format.`,
```
Just while I'm here... I don't like referring to current, existing things as "new" because that doesn't age well. At some point, this isn't "new", it's just the way it is. This is more future-proof.